src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -132,11 +116,42 @@ public Test BuildFrom(FixtureMethod method, Test suite)
                     tests.Add(test); 
             }
 
-            return hasBuildersSpecified && method.Method.GetParameters().Length > 0 || tests.Count > 0
-                ? BuildParameterizedMethodSuite(method, tests)
+            return builders.Count > 0 && method.Method.GetParameters().Length > 0 || tests.Count > 0 
+                ? BuildParameterizedMethodSuite(method, tests) 
```

Comment:
I wouldn't have asked if I wasn't requesting other changes, but while you're at it, could you also remove the spaces being added to the ends of these two lines? Visual Studio hides this from you (without an [extension](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer)), but GitHub and the git CLI don't look as clean.